### ISSUE TITLE
source guild-env depends on TERM environment variable

### ISSUE THREAD

**Issue by Jim-Holmstroem** [guildai/guildai#131]
Unable to run `source guild-env` when `TERM` is not set (for example in a CI environment).

```
TERM environment variable not set.
```
https://github.com/codewell/ml-workflow/pull/93/checks?check_run_id=467526548
under "install dependencies"

setting `TERM=dummy` did not help (unknown terminal type),
`TERM=screen` did work.

This however doesn't have a non-negative return code when you run it locally, but it does when run under github actions:
```
$  TERM="dummy" source guild-env
'dummy': unknown terminal type.
...
```

The vanilla venv
```
TERM="" source venv/bin/activate
```
doesn't depend on TERM being set

not super sure but I think it still works when you specify a recognized but not installed TERM

I have some guess that this is related to that the screen resets when you jump into the environment

**Comment by gar1t** [comment]
Very late reply - but thank you for the report! This is fixed in master and will be available in the next release.

**Comment by gar1t** [comment]
The fix is now generally available in 0.7.0. If this is still an issue, please re-open and we'll sort it out. Thanks again for the report!
